src/floating-action-button.jsx

```diff
@@ -52,15 +52,16 @@ const FloatingActionButton = React.createClass({
     onTouchStart: React.PropTypes.func,
     secondary: React.PropTypes.bool,
     style: React.PropTypes.object,
+    zDepth: React.PropTypes.number,
   },
 
   getInitialState() {
-    let zDepth = this.props.disabled ? 0 : 2;
+    let zDepth = this.props.zDepth ? this.props.zDepth : 2;
     return {
       hovered: false,
       initialZDepth: zDepth,
       touch: false,
-      zDepth: zDepth,
+      zDepth: this.props.disabled ? 0 : zDepth,
```

Comment:
Yes, this should use `getZDepth()`. I was just updating the existing code -- existing code didn't use `getZDepth()`.